lms/templates/ccx/grading_policy.html

```diff
@@ -3,8 +3,8 @@
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
     <i class="msg-icon icon-warning-sign"></i>
-    <div class="msg-content">
-      <h3 class="title">${_("WARNING")}</h3>
+    <div class="msg-content" tabindex="0">
```

Comment:
Thanks for your feedback! I am new to accessibility work and am still trying to figure out how to achieve the right results. In this case, we have a warning message that applies to the form. Can you recommend what would be the most appropriate way for it to be read out to screen reader users?

![screen shot 2015-05-12 at 5 07 52 pm](https://cloud.githubusercontent.com/assets/10431250/7586789/eaf968b4-f8c9-11e4-8870-9e9ae26915ad.png)